tests/snmp/test_snmp_phy_entity.py

```diff
@@ -74,20 +75,12 @@
     'voltage': ('Voltage', 4, SENSOR_TYPE_VOLTAGE),
 }
 
-XCVR_SENSOR_OID_LIST = [SENSOR_TYPE_TEMP,
-                        SENSOR_TYPE_PORT_TX_POWER + 1,
-                        SENSOR_TYPE_PORT_TX_POWER + 2,
-                        SENSOR_TYPE_PORT_TX_POWER + 3,
-                        SENSOR_TYPE_PORT_TX_POWER + 4,
-                        SENSOR_TYPE_PORT_RX_POWER + 1,
-                        SENSOR_TYPE_PORT_RX_POWER + 2,
-                        SENSOR_TYPE_PORT_RX_POWER + 3,
-                        SENSOR_TYPE_PORT_RX_POWER + 4,
-                        SENSOR_TYPE_PORT_TX_BIAS + 1,
-                        SENSOR_TYPE_PORT_TX_BIAS + 2,
-                        SENSOR_TYPE_PORT_TX_BIAS + 3,
-                        SENSOR_TYPE_PORT_TX_BIAS + 4,
-                        SENSOR_TYPE_VOLTAGE]
+XCVR_SENSOR_PATTERN = {
+    'temperature': {'sort_factor': 0, 'oid_base': SENSOR_TYPE_TEMP, 'extract_line_number': False},
+    'voltage': {'sort_factor': 9000, 'oid_base': SENSOR_TYPE_VOLTAGE, 'extract_line_number': False},
```

Comment:
>9000 [](start = 31, length = 4)

How do you choose this number? If it is based on other code, could you provide some code comment to refer that code?